src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxCppCompileStep.cs

```diff
@@ -79,10 +79,10 @@ private void InitializeArgs(NativeCompileSettings config)
             // Add Stubs
             argsList.Add(Path.Combine(config.AppDepSDKPath, "CPPSdk/ubuntu.14.04/lxstubs.cpp"));
```

Comment:
BTW: `lxstubs.cpp` should be removed from here, and its content linked into `libbootstrapper` - where it can gradually disappear as the libraries are ported.